diff --git a/cmake/AWSSDKConfig.cmake b/cmake/AWSSDKConfig.cmake
index ba9183f543..a2c29f1491 100644
--- a/cmake/AWSSDKConfig.cmake
+++ b/cmake/AWSSDKConfig.cmake
@@ -312,5 +312,18 @@ if (AWSSDK_FIND_COMPONENTS)
     endforeach()
 endif()
 
+find_library(LIB_CURL curl)
+if(LIB_CURL)
+  list(APPEND AWSSDK_LINK_LIBRARIES ${LIB_CURL})
+endif()
+find_library(LIB_SSL ssl)
+if(LIB_SSL)
+  list(APPEND AWSSDK_LINK_LIBRARIES ${LIB_SSL})
+endif()
+find_library(LIB_CRYPTO crypto)
+if(LIB_CRYPTO)
+  list(APPEND AWSSDK_LINK_LIBRARIES ${LIB_CRYPTO})
+endif()
+
 set(AWSSDK_INCLUDE_DIRS ${AWSSDK_INCLUDE_DIR})
 set(AWSSDK_LIBRARIES ${AWSSDK_LINK_LIBRARIES})
