--- a/build/generic/Makefile
+++ b/build/generic/Makefile
@@ -149,21 +149,6 @@
 	@$(INSTALL) -d $(DESTDIR)$(libdir)
 	@echo "  I: $(libdir)/$(STATIC_LIB)"
 	@$(INSTALL) -m 644 $(BUILD_DIR)/$(STATIC_LIB) $(DESTDIR)$(libdir)/$(STATIC_LIB)
-ifeq ($(SHARED_EXTENSION),dll)
-	@echo "  I: $(libdir)/$(SHARED_LIB).a"
-	@$(INSTALL) -m 644 $(BUILD_DIR)/$(SHARED_LIB).a $(DESTDIR)$(libdir)/$(SHARED_LIB).a
-	@echo "  D: $(bindir)"
-	@$(INSTALL) -d $(DESTDIR)$(bindir)
-	@echo "  I: $(bindir)/$(SHARED_LIB)"
-	@$(INSTALL) -m 755 $(BUILD_DIR)/$(SHARED_LIB) $(DESTDIR)$(bindir)/$(SHARED_LIB)
-else
-	@echo "  I: $(libdir)/$(SHARED_LIB)"
-	@$(INSTALL) -m 644 $(BUILD_DIR)/$(SHARED_LIB) $(DESTDIR)$(libdir)/$(SHARED_LIB)
-	@test -z "$(SO_API_MAJOR_LINK)" || \
-		$(LN_S) $(SHARED_LIB) $(DESTDIR)$(libdir)/$(SO_API_MAJOR_LINK)
-	@test -z "$(SO_LINK)" || \
-		$(LN_S) $(SHARED_LIB) $(DESTDIR)$(libdir)/$(SO_LINK)
-endif

 #-----------------------------------------------------------------------------
 # Platorm specific file -- dumb rules for people executing make before
